wit/wasi-nn.wit

```diff
@@ -0,0 +1,100 @@
+package wasi:nn
+
+world inference {
+    import tensor
+    import graph
+    import execution
+    import errors
+
+    export run: func()-> result<_,string>
```

Comment:
A world definition that exports a `run` like this is meant for a program that imports those four interfaces, takes no arguments, and has no other imports. I don't think that is actually a world anyone wants to target.

My expectation is that downstream of this spec, embedders will provide the wasi-nn imports, but the export funcs will be part of some other spec. e.g. if its a CLI program the export will be provided by the wasi:cli/command world, and if its used in a web server the export will be provided by the wasi:http/proxy world.

Reply:
This may have been a stray experimentation error while I was trying to track down what was causing the wit-bindgen failure. This can be removed entirely. 

That said, I wasn't 100% clear on what is going to get exported here for use by other components.  Do the graph and execution interfaces need to be exported so that other components can perform inference?